src/js/tech/html5.js

```diff
@@ -848,6 +848,7 @@ Html5.Events = [
   'play',
   'pause',
   'ratechange',
+  'resize',
```

Comment:
this actually made me think maybe we should rename the dimension resize event on the player?
Some people (I know I have) have done `player.on(Html5.Events, () =>)` but the `resize` won't be the correct one.

I propose we change the player-level resize to `dimensionchange` and bring `resize` through unchanged. @videojs/core-committers thoughts?